src/middlewares/lscacheMiddleware.js

```diff
@@ -13,40 +14,49 @@ export function computeItemCacheKey(params) {
   return objectHash(params);
 }
 
-export default function lscacheMiddleware() {
-  /**
-   * @param  {Object}   params
-   * @param  {Function} callback
-   * @param  {Boolean?}  options.cache
-   * @param  {Boolean?}  options.invalidate
-   * @param  {String?}  options.bucketId
-   */
-  return next => function(params, callback, {cache = false, invalidate = false, bucketId} = {}) {
-    if (!cache) {
-      return next(...arguments);
-    }
-
-    const bucket = bucketId ? lscache.createBucket(bucketId) : lscacheBucket;
-    const itemKey = computeItemCacheKey(params);
-    if (!invalidate) {
-      const itemValue = bucket.get(itemKey);
-      if (itemValue) {
-        callback(null, itemValue);
-        return false;
+/**
+ * @param  {?Array<{controllerId, operationId}>} cachedOperations
+ * @return {Middleware}
+ */
+export default function(
+  cachedOperations = []
+) {
+  return function lscacheMiddleware({controllerId, operationId} = {}) {
+    /**
+     * @param  {Object}   params
+     * @param  {Function} callback
+     * @param  {Boolean?}  options.cache
+     * @param  {Boolean?}  options.invalidate
+     * @param  {String?}  options.bucketId
+     */
+    return next => function(params, callback, {cache = false, invalidate = false, bucketId} = {}) {
+      const cachedOperation = find(cachedOperations, co => co.controllerId === controllerId && co.operationId === operationId);
+      if (!cachedOperation && !cache) {
```

Comment:
hum
cachedOperations are operations cached **by default** but then cache can override it